src/i18n/translations/jp/translation.json

```diff
@@ -1,12 +1,18 @@
 {
   "404": {
     "statusCode": "404",
-    "errorMessage": "入力したURLは間違っているか、ページが削除された可能性があります。<2>ホームに戻る</2>"
+    "errorMessage": "入力したURLは間違っているか、ページが削除された可能性があります。&lt;2&gt;ホームに戻る&lt;/2&gt;"
```

Comment:
we don't want the brackets escaped do we?

Reply:
oh my goodness, nope, i'll fix those up

Reply:
ah okay. looks like there's a couple instances of that happening